### PR TITLE
Add flag to work with transactions

### PR DESCRIPTION
**Pull request by EthanSosivio**
Have a flag to work with transaction when creation new client.

### CODE REVIEW

File: conn.go [hunk 1]
```diff
@@ -158,6 +162,14 @@ func (conn *Connection) SetConsistencyLevel(levelDesired string) error {
 	return errors.New(fmt.Sprintf("unknown consistency level: %s", levelDesired))
 }
 
+func (conn *Connection) SetExecutionWithTansaction(state bool) error {
```
Comment (otoolep) [inline conn.go:165]: Tansaction -> Transaction

File: conn.go [hunk 2]
```diff
@@ -282,6 +294,9 @@ func (conn *Connection) initConnection(url string) error {
 		}
 	}
 
+	//deafult transaction state
```
Comment (otoolep) [inline conn.go:297]: deafult -> Default.

File: conn.go (hunk 1 above)
Comment (EthanSosivio) [reply conn.go]: fixed

File: conn.go (hunk 2 above)
Comment (EthanSosivio) [reply conn.go]: fixed

**Comment by EthanSosivio** [comment]
Have a look now :)

**Comment by otoolep** [comment]
Thanks for your contribution.

**Pull request merged**